### DISCUSSION TITLE
附录标题编号

### DISCUSSION THREAD

**Discussion by jqc25z** [sjtug/SJTUThesis#988] [提问] [answered]
附录word版要求编号为1，2，3...而模板里是ABC
![8e20aea96718e7dba9f0d8a09485c15](https://github.com/sjtug/SJTUThesis/assets/112072840/b764d486-52ec-4d1b-b464-9f4a3896ec1b)
是否可以统一修改成123

**Comment by LogCreative** [comment, marked as answer]
duplicate https://github.com/sjtug/SJTUThesis/discussions/934#discussioncomment-6170645
